### ISSUE TITLE
Failed to get image dimensions (Firefox)

### ISSUE THREAD

**Issue by werru** [tinymce/tinymce#5777]
**What is the current behavior? Describe the bug**
Then i insert a link with image like https://sun9-12.userapi.com/c856520/v856520397/1b5dda/tOF8j88RDVk.jpg i get Failed to get image dimensions for: https://sun9-12.userapi.com/c856520/v856520397/1b5dda/tOF8j88RDVk.jpg in console log
This image is from vk.com social network CDN
[image with error](https://i.imgur.com/ku238gN.png)

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**
go to https://www.tiny.cloud/docs/demo/full-featured/ and try to insert image https://sun9-12.userapi.com/c856520/v856520397/1b5dda/tOF8j88RDVk.jpg
**What is the expected behavior?**
To see image

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
5.3.1 prod

**Comment by DNWEIJ** [comment]
https://www.tiny.cloud/docs/demo/full-featured/   selected INSERT and IMAGE...  then paste url of the image. 
Just tried on FF:  working  (version 76.0.1)
I updated FF to 77.01 and it is still working.
Nice space ship :-)
Windows 10.

**Comment by werru** [comment]
Yes, I confirm. All things work in Windows. And doesn't in kUbuntu Linux

**Comment by lincolndennis** [comment]
Can confirm image inserts as expected, with dimension information, in Firefox 78.0.1 running on GalliumOS (which is built on Xubuntu)

**Comment by werru** [comment]
> Can confirm image inserts as expected, with dimension information, in Firefox 78.0.1 running on GalliumOS (which is built on Xubuntu)

Just to clarify. Did you insert https://sun9-12.userapi.com/c856520/v856520397/1b5dda/tOF8j88RDVk.jpg ?

**Comment by dybxin** [comment]
i got the same error, failed to get image dimensions.

**Comment by dybxin** [comment]
> > Can confirm image inserts as expected, with dimension information, in Firefox 78.0.1 running on GalliumOS (which is built on Xubuntu)
> 
> Just to clarify. Did you insert https://sun9-12.userapi.com/c856520/v856520397/1b5dda/tOF8j88RDVk.jpg ?

after i try it, and i can got  the image dimension information too.

**Comment by dybxin** [comment]
> Can confirm image inserts as expected, with dimension information, in Firefox 78.0.1 running on GalliumOS (which is built on Xubuntu)

i think copy image link into 'General--> Source' can get image dimension.  but through "Upload" can not get image dimension sometimes.

**Comment by niemal** [comment]
> i think copy image link into 'General--> Source' can get image dimension. but through "Upload" can not get image dimension sometimes.

I can confirm I am experiencing the exact same behavior.

**Comment by tiny-ci-servers** [comment]
This issue was closed because it has been stalled for 30 days with no activity.
